ntfy/backends/insta.py

```diff
@@ -0,0 +1,23 @@
+import sys
+import logging
+from instapush import App
+
+def notify(title, message, event_name, appid, secret, trackers, retcode=None):
+    logger = logging.getLogger(__name__)
+    msgs = message.split(":")
+
+    if len(msgs) != len(trackers):
+        logger.error("Wrong number of messages! There are {} trackers so you have to provide {} messages. Remember to separate eah message with ':' (example: send \"msg1:msg2\")".format(len(trackers), len(trackers)))
+        sys.exit(1)
+
+    to_send = {}
+
+    for tracker, msg in zip(trackers, msgs):
+        to_send[tracker] = msg
+
+    app = App(appid=appid, secret=secret) 
+    res = app.notify(event_name=event_name,trackers=to_send)
+
+    if res["status"] != 200:
+        logger.error(res["msg"])
+        sys.exit(1)
```

Comment:
ntfy will print an error if it can't send a notification through your backend if you raise an exception. So no need to do `sys.exit(1)` as ntfy will try to send message to other backend for which user has registered.